src/css3/parser_config.rs

```diff
@@ -0,0 +1,32 @@
+use crate::css3::location::Location;
+
+/// Context defines how the data needs to be parsed
+pub enum Context {
```

Comment:
A parser config defines how a parser should behave. What is the context in which to parse the bytestream (is it a stylesheet, is it inline data etc), and we can decide what to do with errors.